tests/pkg/tflite-micro/Makefile.ci

```diff
@@ -1,109 +1,29 @@
 BOARD_INSUFFICIENT_MEMORY := \
-    airfy-beacon \
```

Comment:
This is removing too many boards to be true. I doubt all those removed boards have enough memory. nrf51dk has only 8kB of RAM.

Reply:
I generated it through the `RIOT/dist/tools/insufficient_memory/create_makefile.ci.sh` script, but i must admit i don't have that board to test it on. Running `BOARD=airfy-beacon make -C tests/pkg/tflite-micro/` it builds fine, but I can't actually run it. The closest i have access to is a microbit, and this compiles and works on that